modules/programs/rbw.nix

```diff
@@ -41,9 +41,9 @@ let
         };
 
         pinentry = mkOption {
```

Comment:
Maybe worth renaming this as well to `pinentryPackage`. Then could also add a `mkRemovedOptionModule`?

Reply:
It's used directly by the config file, which expects `pinentry`.

Reply:
Ah, right. Fair enough!